releasenotes-builder/src/test/java/com/github/checkstyle/internal/AbstractReleaseNotesTestSupport.java

```diff
@@ -177,12 +178,34 @@ protected void runMainAndAssertReturnCode(int expectedExitCode, String... argume
         catch (Exception exception) {
             fail(String.format("Unexpected exception: %s", exception));
         }
-        if (expectedExitCode == 0) {
-            verify(mock, never()).exit(expectedExitCode);
-        }
-        else {
+        finally {
             verify(mock).exit(expectedExitCode);
         }
     }
 
+    /**
+     * Helper method to call {@code runMainAndAssertReturnCode}
+     * with some default required arguments.
+     *
+     * @param expectedExitCode the expected exit code to verify
+     * @param arguments the command line arguments
+     */
+    protected void runMainContentGenerationAndAssertReturnCode(int expectedExitCode,
+                                                         String... arguments) {
+        final ArrayList<String> listOfArgs = new ArrayList<>(Arrays.asList(arguments));
+        listOfArgs.addAll(
+            Arrays.asList(
+                "-localRepoPath", getTempFolder().getAbsolutePath(),
+                "-remoteRepoPath", "checkstyle/checkstyle",
+                "-startRef", "12345678",
+                "-outputLocation", getTempFolder().getAbsolutePath(),
+                "-githubAuthToken", "TOKEN"
+            )
+        );
+        String[] arrOfArgs = new String[listOfArgs.size()];
+        arrOfArgs = listOfArgs.toArray(arrOfArgs);
```

Comment:
Not a big fan of what I did here, but I see no other way of appending elements to `arguments`. Open to suggestions if you see a better way. 

Shame that Java does not have a spread operator like JavaScript...

Reply:
You could do:
````
    String[] result = Arrays.copyOf(array1, array1.length + array2.length);
    System.arraycopy(array2, 0, result, array1.length, array2.length);
    return result;
````
or
````
    String[] result = new String[len1 + len2];
    System.arraycopy(array1, 0, result, 0, len1);
    System.arraycopy(array2, 0, result, len1, len2);
````

Reply:
you could also do something like:
```
        final List  a = List.of(...); // your args
        a.addAll(List.of(arguments)); // args passed in
        runMainAndAssertReturnCode(expectedExitCode, a.toArray(new String[0]));

```

Reply:
Done. I used Nick's suggestion.